src/elements/content-sidebar/SidebarContentInsights.scss

```diff
@@ -0,0 +1,12 @@
+.bcs-SidebarContentInsights {
+    .ContentAnalyticsErrorState {
+        svg {
+            margin-top: 25px;
+        }
+    }
+
+    .ContentAnalyticsErrorState-text,
+    .ContentAnalyticsErrorState-text--permission {
```

Comment:
Modifier classes should be added to elements alongside the base class. Does the element affected in this case not have `ContentAnalyticsErrorState-text` applied?